### PR TITLE
[Merged by Bors] - chore(LinearAlgebra): Fintype -> Finite

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/StdBasis.lean [hunk 1]
```diff
@@ -321,7 +322,7 @@ end Module
 
 namespace Matrix
 
-variable (R : Type*) (m n : Type*) [Fintype m] [Fintype n] [Semiring R]
+variable (R : Type*) (m n : Type*) [Fintype m] [Finite n] [Semiring R]
```
Comment (ADedecker) [inline Mathlib/LinearAlgebra/StdBasis.lean:325]: What about `m`?

**Review by ADedecker — COMMENTED** [review]
bors d+

File: Mathlib/LinearAlgebra/StdBasis.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/LinearAlgebra/StdBasis.lean:325]: We can make it `Finite` but this requires more work. Currently it's used in a definition.

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**